ddp-workspace/projects/ddp-sdk/src/lib/services/picklistSortingPolicy.service.ts

```diff
@@ -0,0 +1,49 @@
+import { ActivityPicklistNormalizedGroup, ActivityPicklistOption } from 'ddp-sdk';
+import { SortOrder } from './sortOrder';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root',
```

Comment:
I believe `{provideIn: 'root'}` is not needed because the service is already provided in ddp.module

Reply:
I'd recommend to not remove provideIn, it's recommended way to inject services in applications
https://angular.io/guide/providers
for example it improves tree-shaking
it's better to not provide it in sdk module in this case

Reply:
On the one hand it makes sense,
on the other hand all services in `ddp-sdk` are provided in the module only.